sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AnalyzeBatchActionsResult.java

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.models;
+
+import com.azure.ai.textanalytics.implementation.AnalyzeBatchActionsResultPropertiesHelper;
+import com.azure.core.util.IterableStream;
+
+/**
+ * The {@link AnalyzeBatchActionsResult} model.
+ */
+public final class AnalyzeBatchActionsResult {
```

Comment:
Missing annotation here and other model classes.

Reply:
We have ***PropertyHelper to set the properties internally. So the Immutable annotation can't be used.